cypress/e2e/clue/branch/teacher_tests/teacher_chat_spec.js

```diff
@@ -299,7 +325,8 @@ context('Chat Panel', () => {
       chatPanel.verifyCommentThreadContains("This is a teacher8 document comment");
       // Teacher 7 tile comment
       cy.clickProblemResourceTile('introduction');
+      cy.wait(2000);
```

Comment:
Yeah. ☹️ I think if the tests cleaning up after themselves would help.  Since they don't, sometimes the list of comments is really long. I also think there's some improvements to be made about knowing whether we're in a loading state or not that *might* help. Is this the kind of followup it would be good to track in a story?